doc/api/http2.md

```diff
@@ -2977,6 +2977,16 @@ message) to the response.
 Attempting to set a header field name or value that contains invalid characters
 will result in a [`TypeError`][] being thrown.
 
+#### response.bytesWritten
+<!-- YAML
+added: CHANGEME
+-->
+
+* {integer}
+
+Integer value that indicates how many bytes has been written to
+the response body (not the amount flushed).
```

Comment:
I think it would be better to refer to `Writable.bytesWritten` here? Could we do that? How?